riscv32imfc-unknown-none-elf.json

```diff
@@ -5,7 +5,7 @@
   "eh-frame-header": false,
   "emit-debug-gdb-scripts": false,
   "executables": true,
-  "features": "+m,+f,+c",
+  "features": "+m,+a,+f,+c",
```

Comment:
The core doesn't support atomics afaik. I think there is some confusion about that